package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "cagov-benefits-screener",
+    "name": "cagov-unemployment",
```

Comment:
i don't think this affects anything since we aren't publishing our package https://docs.npmjs.com/files/package.json